cnudie/util.py

```diff
@@ -414,6 +417,140 @@ def find_changed_component(
     )
 
 
+def format_component_diff(
+    component_diff: ComponentDiff,
+    delivery_dashboard_url_view_diff: str,
```

Comment:
Adjust type hint to allow `None` as value + preferably also set it as default value (same for `delivery_dashboard_url`).